packages/rollup-plugin-webbundle/README.md

```diff
@@ -273,6 +282,10 @@ https://github.com/WICG/isolated-web-apps.
 
 ## Release Notes
 
+### v0.2.0
+
+- Add support for the v2 integrity block format.
```

Comment:
nit: can you add a very short 1 sentence description here too what that means?

Reply:
also both of these comments apply to the other readme too

Reply:
Done!